Bonsai.Tinkerforge/BrickletDeviceNameConverter.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using Bonsai.Expressions;
+using Tinkerforge;
+using System.Threading;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Bonsai.Tinkerforge
+{
+    internal class BrickletDeviceNameConverter : TypeConverter
+    {
+        public Dictionary<string, TinkerforgeHelpers.DeviceData> devices;
```

Comment:
This cache could be initialized in the `ConvertTo` and `ConvertFrom` methods, and refreshed in `GetStandardValues`.